tox.ini

```diff
@@ -30,19 +29,18 @@ deps =
   setuptools >= 40.4.3
   wheel
 commands =
-  ansibledevel: sh -c "{envpython} -m pip show ansible | >/dev/null grep '^Version: 2.10.0$'"
-  ansibledevel: sh -c "{envpython} -m pip show ansible-base | >/dev/null grep '^Version: 2.10.0.dev0$'"
+  # safety measure to assure we do not accidentaly run tests with broken dependencies
+  python -m pip check
   # Collections outside of ACD that config/routing.yml points to in Core:
-  ansibledevel: ansible-galaxy collection install --collections-path "{envtmpdir}" git+https://github.com/ansible-collections/amazon.aws
-  ansibledevel: ansible-galaxy collection install --collections-path "{envtmpdir}" --pre chocolatey.chocolatey
+  ansibledevel: ansible-galaxy collection install --collections-path "{envtmpdir}" amazon.aws
+  ansibledevel: ansible-galaxy collection install --collections-path "{envtmpdir}" chocolatey.chocolatey
   {envpython} -m pytest \
   --cov "{envsitepackagesdir}/ansiblelint" \
   --junitxml "{toxworkdir}/junit.{envname}.xml" \
   {posargs:}
 install_command =
   {envpython} -m \
     pip install \
-    --unstable-feature=resolver \
```

Comment:
This broke zuul jobs as this feature was not supported by the pip version present on these hosts. That is a too-new feature to use.

Reply:
It's not because of pip. It's because of virtualenv (which effectively pulls-in a certain pip version)